### ISSUE TITLE
weighting unit test (Trac #1239)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1233]
since weighting is pure python our code coverage report does not give info, but there are two python scripts in resources/test

compare_oneweight.py appears to be a useful unit test for nugen weighting but the assert is commented out 
corsika_weight_calculator.py wants cli input and does not appear to be a unit test

Migrated from https://code.icecube.wisc.edu/ticket/1239
```json
{
    "status": "closed", 
    "changetime": "2016-03-18T21:14:03", 
    "description": "since weighting is pure python our code coverage report does not give info, but there are two python scripts in resources/test\n\ncompare_oneweight.py appears to be a useful unit test for nugen weighting but the assert is commented out \ncorsika_weight_calculator.py wants cli input and does not appear to be a unit test", 
    "reporter": "kjmeagher", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1458335643235016", 
    "component": "combo reconstruction", 
    "summary": "weighting unit test", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2015-08-20T08:32:45", 
    "milestone": "", 
    "owner": "jvansanten", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150824T17:28:31: 
* olivas changed milestone from "offline V15-08-00" to "IceSim 4.2"
* olivas commented:

> weighting isn't part of offline.

**Comment by icecube-trac** [comment]
Trac update at 20150824T17:32:07: olivas changed component from "icecube offline" to "icerec"

**Comment by icecube-trac** [comment]
Trac update at 20150826T11:23:20: 
* jvansanten changed resolution from "" to "fixed"
* jvansanten commented:

> In changeset 136880:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="136880"
> Run automated tests. These query the simprod db, so prepare for the same
> kind of fun as with the old I3Db tests. Closes #1233.
> ```


* jvansanten changed status from "new" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20160318T21:14:03: 
* olivas commented:

> Milestone IceSim 4.2 deleted


* olivas changed milestone from "IceSim 4.2" to ""
